account_invoice_margin/tests/test_account_invoice_margin.py

```diff
@@ -0,0 +1,94 @@
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields
+from odoo.tests.common import Form, tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestAccountInvoiceMargin(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountInvoiceMargin, cls).setUpClass()
```

Comment:
Apply PEP 3135 (28-Apr-2007) to "super()."
```suggestion
        super().setUpClass()
```